dev/ProgressRing/ProgressRingLoading.cpp

```diff
@@ -0,0 +1,362 @@
+﻿// This file was auto-generated then been modified to update animation visuals with inherited visual properties
+
+#include "pch.h"
+#include "common.h"
+#include "ProgressRingLoading.h"
+#include "WindowsNumerics.h"
+
+static bool IsRuntimeCompatible()
+{
+    if (!winrt::Windows::Foundation::Metadata::ApiInformation::IsTypePresent(L"Windows.UI.Composition.CompositionGeometricClip"))
+    {
+        return false;
+    }
+    return true;
+}
+
+class ProgressRingAnimatedVisual :
```

Comment:
We can remove ProgressRingLoading.cpp/.h, since we have the new file ProgressRingIndeterminate to replace it.